.circleci/config.yml

```diff
@@ -126,7 +126,7 @@ jobs:
     <<: *opam-boot-template
     environment:
       <<: *envvars
-      EXTRA_OPAM: "ocamlgraph elpi"
+      EXTRA_OPAM: "ocamlgraph ppx_tools_versioned ppx_deriving ocaml-migrate-parsetree ounit elpi"
```

Comment:
This does not look good and is the sign of uncorrectly solved conflicts. You shouldn't need "ppx_tools_versioned ppx_deriving ocaml-migrate-parsetree" at all in this branch.

Reply:
And, FWIW, "crept" is preferred to "creeped" with one exception: http://grammarist.com/usage/creeped-crept/

Reply:
Thanks for the language tips!